app/assets/javascripts/application.js

```diff
@@ -12,6 +12,7 @@
 //= require twitter/bootstrap
 //= require dataTables/jquery.dataTables
 //= require dataTables/bootstrap/3/jquery.dataTables.bootstrap
+//= require dataTables/extras/dataTables.colVis
```

Comment:
Isn't this extension deprecated? https://datatables.net/extensions/colvis/

Reply:
Yes, you are right. As per the `readMe file`, I was in the assuption that the latest option `buttons` was not available in the `gem`, but checking the code I could see that there is the `button extra`. I will verify and will do the corresponding changes.

Reply:
@pierre, apparently the `gem` does not support the `column visibility module for Buttons`.  I have to implement the functionality using the dataTable API.

Reply:
I see - let's forget about it then for now and use `colVis`.

Eventually, we should review our DataTables integration though (I've created https://github.com/killbill/killbill-admin-ui/issues/175 in the backlog).